claim-db-worker/app/web/layout.tsx

```
@@ -0,0 +1,342 @@
+                fill="white"
+              />
+            </svg>
+          </div>
+          <p className="mt-4 text-lg text-muted text-center px-4">
+            Your temporary database has expired or has been claimed. Please
+            create a new one.
+          </p>
+          <button
+            onClick={confirmCreateNewDatabase}
+            className="px-4 py-2 bg-button-blue font-bold mt-6 hover:bg-button-blue-hover text-white rounded transition-colors"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)